client/influxdb.go

```diff
@@ -112,13 +99,21 @@ func (c *Client) Write(writes ...Write) (*Results, error) {
 	defer resp.Body.Close()
 
 	var results Results
-	dec := json.NewDecoder(resp.Body)
-	dec.UseNumber()
-	err = dec.Decode(&results)
+	body, _ := ioutil.ReadAll(resp.Body)
+	if len(body) > 0 {
```

Comment:
@evanphx yeah, that's a good point.  I looked at it twice when I did my code reviews and felt it kind of smelled.  Really I should just check if the error is `EOF`, as that is the normal case.  If it is, move on.  If not, throw an error.